build.sh

```diff
@@ -231,19 +231,48 @@ function BuildBoost()
     fi
 }
 
-function BuildRocketMQClient()
+BuildRocketMQClient()
 {
     cd ${build_dir}
     cmake ..
-    make
+    make -j8
     if [ $? -ne 0 ];then
         exit 1
     fi        
     #sudo make install
     PackageRocketMQStatic
 }
 
-function PackageRocketMQStatic()
+BuildGoogleTest()
```

Comment:
also need consider user can copy source file to rocektmq-client-cpp when can't get internet